CUJExecutor/manipulateProfilesAndTest

```diff
@@ -14,60 +14,123 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-INPUT_FILE=${1:-input.txt}
-INCLUDE_MEASURED_CUJ=${2?"ERROR: need to specify whether the measured CUJ should be included in the profile"}
-FILE_NAME="${INPUT_FILE%.*}"
-PRE_CUJ=$(sed '1q;d' $INPUT_FILE)
-POST_CUJ=$(sed '2q;d' $INPUT_FILE)
-RECORD=$(sed '4q;d' $INPUT_FILE)
-
 
+##compiles the app according to the COMPILATION_FLAG and then measures the cuj specified by the INPUT_FILE 
+##stores a command/output summary, a short data summary, the profile used to compile the app (if applicable) 
+##and a recording of the median run 
```

Comment:
This file comment is insufficient.  It needs to describe the behavior, the inputs, the outputs, and an example invocation.